src/formatter/builder.ts

```diff
@@ -99,28 +105,39 @@ const FormatterBuilder = {
     let Syntax = JavascriptSnippetSyntax
     if (doesHaveValue(snippetSyntax)) {
       const fullSyntaxPath = path.resolve(cwd, snippetSyntax)
-      Syntax = require(fullSyntaxPath) // eslint-disable-line @typescript-eslint/no-var-requires
+      Syntax = await importer(pathToFileURL(fullSyntaxPath))
+      Syntax = FormatterBuilder.resolveConstructor(Syntax)
```

Comment:
```suggestion
      const syntaxImporter = await importer(pathToFileURL(fullSyntaxPath))
      Syntax = FormatterBuilder.resolveConstructor(syntaxImporter)
```
Or something like that?
Seeing two times `Syntax = ` puzzled me actually

Reply:
Agreed - fixed by inlining the import